packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -26,10 +27,20 @@ const execAppOptions = getDefaultExecSyncOptions();
 const packageManager = getPackageManager();
 
 describe('Create new sdk command', () => {
-  beforeEach(() => {
```

Comment:
can't you revert this change ? it seems the beforeEach should be working just fine and you are no longer using `_sdkFolderIndex`

Reply:
of course. It's not needed anymore